kernels/volk/volk_32f_s32f_convert_32i.h

```diff
@@ -241,14 +241,17 @@ static inline void volk_32f_s32f_convert_32i_generic(int32_t* outputVector,
     float min_val = INT_MIN;
     float max_val = INT_MAX;
     float r;
```

Comment:
This implementation would benefit from quite a bit of clean-up. Would you be willing to include this in your PR?

I suggest to move the loop variable declaration `number` into the for loop.
The `float r` declaration could go into the loop as well. `min_val` and `max_val` should be `const`. 
The `int s;` declaration is only necessary inside the loop. Thus, I suspect it would be better to move it into this scope.

Reply:
Ok, I can do that.

I agree, having two copies of the same code is a bad idea (which caused me to miss fixing one of the copies in the previous attempt); one should call the other.